codespell_lib/tests/test_dictionary.py

```diff
@@ -75,7 +75,7 @@ def _check_aspell(phrase, msg, in_aspell, fname):
     elif in_aspell is False:
         languages = supported_languages
     elif in_aspell is True:
```

Comment:
As both `in_aspell is False` and `in_aspell is True` now result in `languages = supported_languages`, you could combine them into a single conditional?
i.e.
```python3
    if in_aspell is None:
        return  # don't check
    elif in_aspell is False or in_aspell is True:
        languages = supported_languages
    else:
        languages = in_aspell
```